### ISSUE TITLE
Web documentation wrong

### ISSUE THREAD

**Issue by hoyleb** [fchollet/keras#1660]
Hi team, great job with Keras.

On http://keras.io/examples/

It appears (to me) as if the examples are missing the .core and .convolutional in the "from" portion of the import

--current
from keras.layers import Dense, Dropout, Activation, Flatten
from keras.layers import Convolution2D, MaxPooling2D

--should be
from keras.layers.core import Dense, Dropout, Activation, Flatten
from keras.layers.convolutional import Convolution2D, MaxPooling2D

Perhaps I've been screwing something up.

Thanks,

Ben

**Comment by awentzonline** [comment]
The keras.layers module imports all of those classes for your convenience, so either way works.
